Dockerfile

```diff
@@ -9,14 +9,77 @@ ENV LDAP_SERVER_URL ldap://${LDAP_HOST}:${LDAP_PORT}
 ENV FUSIONDIRECTORY_VERSION 1.0.8.2
 ENV FUSIONDIRECTORY_DEB_PKG_VERSION 1.0.8.2-5+deb8u1
 ENV LDAP_ROOT Manager
+ENV LDAP_TLS TRUE
+ENV TLS_REQCERT never
+
+# Variables FusionDirectory plugins (FALSE = Not installed)
+ENV PLUGIN_ADDRESSBOOK FALSE
+ENV PLUGIN_ALIAS FALSE
+ENV PLUGIN_APACHE2 FALSE
+ENV PLUGIN_APPLICATIONS FALSE
+ENV PLUGIN_ARGONAUT FALSE
```

Comment:
Are these env vars used in FD code? Could you point me to the place where they are used?